src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -126,6 +126,7 @@ protected override void ContextUnloaded(ExecutionContext context)
                 case null:
                     break;
                 case Action action:
+                    Pop();
```

Comment:
Why pop and not `state.NeedCheckReturnValue = CheckReturnType.EnsureIsEmpty;` in `CallFromNativeContract`?